### DISCUSSION TITLE
Is there any way to add styles to a graph?

### DISCUSSION THREAD

**Discussion by partolaaa** [GSI-CS-CO/chart-fx#608] [General]
Hi all :)

I'm currently developing a graph for data analysis and I would like to add styles for grid lines like in this example: docs/pics/ScatterAndBubbleRendererSample2.png

Also, is it possible to display a vertical and horizontal line for y=0 x=0 respectively?

Unfortunately, I didn't find any mention of chart styling in the documentation, so I would appreciate your help!

**Comment by ennerf** [comment]
The screenshot was generated by [ScatterAndBubbleRendererSample](https://github.com/fair-acc/chart-fx/blob/5d5f9363f48228ef528dd5f5b2802f68d1f08ef1/chartfx-samples/src/main/java/io/fair_acc/sample/chart/ScatterAndBubbleRendererSample.java). That being said, #603 will add significantly more styling options and CSS support.
